src/GenCtypes.ml

```diff
@@ -0,0 +1,276 @@
+(* Copyright (c) INRIA and Microsoft Corporation. All rights reserved. *)
+(* Licensed under the Apache 2.0 License. *)
+
+(** Generating Ctypes OCaml bindings for C declarations *)
+
+open CStar
+open CStarToC11
+
+open Migrate_parsetree
+open Migrate_parsetree.Ast_405
+open Migrate_parsetree.Ast_405.Parsetree
+
+open Lexing
+open Pprintast
+open Ast_helper
+open Asttypes
+open Location
+open Longident
+
+
+let migration = Versions.migrate Versions.ocaml_405 Versions.ocaml_current
```

Comment:
So we are moving from the current version to the 4.05 AST. Any reason for 4.05? Can we pick something more recent, e.g. 4.08? (I don't have a strong opinion: just wondering.)

Reply:
I think it's the other way around. We're building a 4.05 AST which we then migrate to the version we're currently using. I think this could in principle work with any version, when I chose 4.05 (both here and in F*) it was because I thought it's the lowest supported version (looking at the README now I see that 4.04 is actually also supported?)